docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+  web:
+    build: . 
+    ports:
+      - "3000:3000"
+    volumes:
+      - .:/usr/src/app
+      - gem_cache:/gems
+  redis:
+    image: redis:latest
+  solr:
+    image: solr:6.6.5
```

Comment:
This application requires at least Solr 7, but runs on 8. This should be the latest stable release.